Documentation/Communicational Patterns/2. Tooling/2.4. ObservedObject.md

```diff
@@ -8,6 +8,12 @@ The `@ObservedObject` wraps an `ObservableObject`. An `ObservableObject` may hav
 
 The lifetime of the `ObservedObject` itself depends on the way it is created. The developer could pass it in from the outside (e.g. from a lower `Component`) as sort of a singleton which lives until the process dies. If the `ObservedObject` is created inside the `Handler` it is recreated every time a new connection is established but lives for the complete lifetime of the `Handler`.
 
+`ObservedObject`s can be globally declared by defining them in the `configuration` computed property of the web service using `Environment(keypath, value)`. This `Configuration` takes as arguments a keypath that is used to retrieve the `ObservableObject` from a property wrapper `@ObservedObject(\.keypath)` and the corresponding value that should be injected. Locally defined `ObservableObject`s don't require the use of a keypath.
```

Comment:
I would suggest to add a small example here that showcases the configuration syntax.